lib/simple/sql.rb

```diff
@@ -26,6 +26,18 @@ module SQL
 
     delegate [:logger, :logger=] => ::Simple::SQL::Logging
 
+    LOCK = 4711
+
+    def locked
+      binding.pry
```

Comment:
The ID of the lock should be passed in from the caller, and not be hardfixed here. This must be an integer

Reply:
Also I think the method should be implemented in the ConnectionAdapter class, dunno